sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -88,6 +88,8 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     "datetime-special.sql",
     "ansi/datetime-special.sql",
     "timestampNTZ/datetime-special.sql",
+    // SPARK-42921
+    "timestampNTZ/datetime-special-ansi.sql",
```

Comment:
ignore in `ThriftServerQueryTestSuite`, same reason as `timestampNTZ/datetime-special.sql`